build.properties

```diff
@@ -64,6 +64,8 @@ tmp.dir = ${project.basedir}/tmp
 # Exported configuration.
 drupal.config.dir = ${project.basedir}/config
 
+# Drupal private files directory (relative to ${project.basedir}).
+drupal.private_files.dir = private
```

Comment:
Let's use the full relative path here, so that we don't need to construct this every time it is used.

But again, we don't need to deal with this on a build level. If people want to use a different path they can manually change their settings file, we don't need to impose anything here.